src/main/java/com/vmtapp/enterprise/TicketController.java

```diff
@@ -82,7 +100,6 @@ public Ticket createTicket(@RequestBody Ticket ticket){
         try {
             newTicket= ticketService.save(ticket);
         } catch (Exception e) {
-
```

Comment:
Never have an empty catch block.  That's throwing away good information about what went wrong.  At the least, put in a proper logging statement.